community/V.gitignore

```diff
@@ -0,0 +1,14 @@
+*
+!*/
+!*.*
```

Comment:
These three lines are problematic and I don't think they should included in a default gitignore configuration. While they are somewhat effective at ignoring binary files on non-windows systems that don't have an extension, they will also ignore other files like Makefiles, and also override any users global gitignore configuration which can lead to engineers accidentally committing secrets or IDE files that they exclude using global gitignore configurations. I've opened a PR proposing they be removed: https://github.com/github/gitignore/pull/3727